### DISCUSSION TITLE
How does CSharp.lua handle Pinvoke?

### DISCUSSION THREAD

**Discussion by ghost** [yanghuan/CSharp.lua#499] [Q&A] [answered]
I'm very curious.

**Comment by yanghuan** [comment]
https://github.com/yanghuan/CSharp.lua/wiki/FAQ_en#7how-do-you-call-or-insert-lua-code-in-your-code

**Reply by ghost** [reply]
You misunderstood me. What I'm talking about is Pinvoke in existing C# code. Long story short, I want to know how CSharp.lua will handle C FFI, not Lua FFI. CSharp.lua will ignored them, or failed to process them? This is what I wanted to know.

**Reply by ghost** [reply, marked as answer]
I tested it myself and got the answer. It will simply ignore Pinvoke code.
